.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: python
 python:
   - "3.8"
+before_install:
+- pip install poetry
 
 install:
-  - pip install -r requirements.txt
+  - poetry install
 
 script:
   - flake8 --ignore=F401
```

Comment:
its strange, cause i think it should works like this
poetry run flake8 --ignore=F401